### PR TITLE
Do not require "experimental" for metrics API

### PR DESCRIPTION
**Pull request by thaJeztah**
This removes the requirement to enable experimental on the daemon to expose the prometheus metrics

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
still draft?

**Comment by thaJeztah** [comment]
Should probably be ok for review; rebased and moved out of "draft"

**Review by tao12345666333 — APPROVED** [review]
LGTM  :+1:

**Comment by thaJeztah** [comment]
ping @ehazlett PTAL; wdyt?

**Comment by thaJeztah** [comment]
(@ehazlett discussing with @tonistiigi to check if the metrics we have currently would conflict with metrics we could add in future, coming from containerd)

File: daemon/metrics_unix.go [hunk 1]
```diff
@@ -28,7 +29,10 @@ func (daemon *Daemon) listenMetricsSock() (string, error) {
 	mux := http.NewServeMux()
 	mux.Handle("/metrics", metrics.Handler())
 	go func() {
-		http.Serve(l, mux)
+		logrus.Infof("metrics API listening on %s", l.Addr())
```
Comment (cpuguy83) [inline daemon/metrics_unix.go:32]: Can we make this a debug log?

**Comment by thaJeztah** [comment]
@cpuguy83 @tonistiigi updated; PTAL

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: daemon/metrics_unix.go [hunk 2]
```diff
@@ -28,7 +29,10 @@ func (daemon *Daemon) listenMetricsSock() (string, error) {
 	mux := http.NewServeMux()
 	mux.Handle("/metrics", metrics.Handler())
 	go func() {
-		http.Serve(l, mux)
+		logrus.Debugf("metrics API listening on %s", l.Addr())
```
Comment (tiborvass) [inline daemon/metrics_unix.go:32]: I guess I don't understand why we have two places (here and in cmd/dockerd) where we start the metrics server.

Comment (cpuguy83) [reply daemon/metrics_unix.go:32]: This is for metrics plugins.

**Review by tiborvass — APPROVED** [review]

**Pull request merged**